tests/testthat/test-read_kallisto.R

```diff
@@ -5,7 +5,8 @@ kallisto_dir <- file.path(dir, "Breast_Cancer_3p_LT/kallisto_txome")
 sce_size <- c(60666, 30339)
 
 test_that("Check that kallisto import works", {
-  sce <- read_kallisto(kallisto_dir)
+  sce <- read_kallisto(kallisto_dir,
```

Comment:
can we add testing the `mapping_tool` and `transcript_type` here too (if we set it?)

Reply:
We don't add any metadata to the kallisto objects, but we could?

Reply:
I think we should... Since the transcript type in `counts` can vary, it would be a nice update to have it in the object, I think.